package/wpewebkit/wpewebkit.mk

```diff
@@ -64,6 +78,42 @@ else
 WPEWEBKIT_CONF_OPTS += -DENABLE_XSLT=OFF
 endif
 
+ifeq ($(BR2_PACKAGE_WPEWEBKIT_LAUNCHER_MINIBROWSER),y)
+WPEWEBKIT_CONF_OPTS += -DENABLE_MINIBROWSER=ON
+else
+WPEWEBKIT_CONF_OPTS += -DENABLE_MINIBROWSER=OFF
+endif
+
+ifeq ($(BR2_PACKAGE_WOFF2),y)
+WPEWEBKIT_CONF_OPTS += -DUSE_WOFF2=ON
+WPEWEBKIT_DEPENDENCIES += woff2
+else
+WPEWEBKIT_CONF_OPTS += -DUSE_WOFF2=OFF
+endif
+
+ifeq ($(BR2_PACKAGE_NINJA),y)
+WPEWEBKIT_CONF_OPTS += -G Ninja
+WPEWEBKIT_DEPENDENCIES += host-ninja
+WPEWEBKIT_MAKE = $(HOST_DIR)/usr/bin/ninja
+endif
+
+define WPEWEBKIT_INSTALL_STAGING_CMDS
+    install -Dm755 $(@D)/bin/WPE{Network,Storage,Web}Process $(STAGING_DIR)/usr/bin/ && \
```

Comment:
Sure. But are you ok with INSTALL_TARGET_CMDS ?

Reply:
Ah, I should have been clearer: None of the `WPEWEBKIT_*_CMDS` should be needed. You can use the [upstream webkitgtk.mk file](https://git.buildroot.net/buildroot/tree/package/webkitgtk/webkitgtk.mk) for inspiration :wink:

Reply:
I would like to have more discussion on this.

1.  If we dont use WPEWEBKIT_*_CMDS then buildroot will use default rule to install. And The default rule uses Development component which will install development files ( header files, pc files etc) which is fine for STAGING install but not for TARGET install
2.  If we use single command like you mentioned above, it will only install development files ( header files, pc files etc) and wont install library binaries and libraries.

Reply:
It's not a problem to use the default commands because Buildroot nowadays automatically cleans up uneeded development files from `$(TARGET_DIR)` at the end of a successful build, using the `target-finalize` Make target, before creating the images — that's why the `webkitgtk.mk` included upstream works fine.